examples/generator-fullstack/.graphqlrc.yml

```diff
@@ -1,63 +1,31 @@
-schema: ./server/src/schema/*.graphql
-documents: ./client/src/graphql/**/*.graphql
+## GraphQL Config Generated by Graphback
```

Comment:
git reset --soft on this and remove it from commit as anything from it should not be commited. Also rebase is needed.